README.md

```diff
@@ -344,6 +344,87 @@ $ cargo install scx_rusty
 
 and `scx_rusty` will be built and installed as `~/.cargo/bin/scx_rusty`.
 
+## Checking scx_stats
+
+- To check the scheduler statistics, use the
```

Comment:
How about something like:

Most schedulers implement `--monitor $INTERVAL` which fetches and shows the detailed statistics from the running scheduler. Note that this does not launch the scheduler. The monitoring instance only shows the statistics. Some schedulers may implement different or multiple monitoring options. Refer to `--help` of each scheduler for details. Most schedulers also accept `--stats $INTERVAL` to print the statistics directly from the scheduling instance.

Reply:
Updated, could you take a look now @htejun

Reply:
Maybe use "statistics" consistently? Schedulers do print out logs like info and warning messages. It just doesn't print out detailed statistics by default.